examples/GML/3.1.1/position-00.gml

```diff
@@ -0,0 +1,5 @@
+<gml:position xmlns:gml="http://www.opengis.net/gml">
```

Comment:
Do GML files need the `<?xml version="1.0" encoding="UTF-8"?>` prefix?

Reply:
No, not necessarily.